cmd/secrets-store-csi-driver/main.go

```diff
@@ -96,5 +98,13 @@ func main() {
 
 func handle() {
 	driver := secretsstore.GetDriver()
-	driver.Run(*driverName, *nodeID, *endpoint, *providerVolumePath, *minProviderVersion)
+	cfg, err := config.GetConfig()
```

Comment:
I recall we had to call `getClient()` to get the latest client to prevent from reading from cache. Is this no longer a concern?

Reply:
https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/client/client.go#L42-L52

```
// New returns a new Client using the provided config and Options.
// The returned client reads *and* writes directly from the server
// (it doesn't use object caches).  It understands how to work with
// normal types (both custom resources and aggregated/built-in resources),
// as well as unstructured types.
```

Reply:
I was thinking about this one https://github.com/kubernetes-sigs/controller-runtime/issues/403
But I dont think we have multiple controllers updating the same objects now.

Reply:
Right, we don't have any update operations on `spc`. There is only `GET` for `spc` and `CREATE` for `spcpodstatus`

Reply:
Adding the links here for future ref 

https://github.com/kubernetes-sigs/controller-runtime/blob/cea989b02ed02254d751ad30471c40bacd5888fc/pkg/client/client_cache.go#L88-L96
    
newResource always creates a new client - https://github.com/kubernetes-sigs/controller-runtime/blob/cea989b02ed02254d751ad30471c40bacd5888fc/pkg/client/client_cache.go#L51-L68